### ISSUE TITLE
linebreaks in multiline cell gets eliminated

### ISSUE THREAD

**Issue by oupala** [omeka-s-modules/CSVImport#181]
I think I'm encoutering an issue which is close to another issue #178.

I am testing a cvs import with the following test file:

```
dcterms:identifier,dcterms:title,dcterms:bibliographicCitation
1503,title string 1,"bib string 1
bib string 2
bib string 3"
1504,title string 2,"bib string 1
bib string 2
bib string 3"
```

The import goes well and when I have a look at the item, I can see that all bibliographic citations are displayed on the same line.

First, the first line break in eliminated, just as in #178 issue. Then the other linebreak are converted into an `<br>` but the br is not rendered as is in the default template (I'm using the default template).

Hence, in the end, the 3 bibliographic citations are all displayed on one line only, due to 2 different bugs.

Any tought?

**Comment by zerocrates** [comment]
I haven't made the release that fixes #178, but it's committed already, so that part should be resolved with the next release.

The issue of the further linebreaks is something I haven't heard of so far so I'm interested in more info on that. Can you send a screenshot (or even better, a live link) showing the issue in the created items? Also, Is this problem happening on the admin and public sides both, or just one?

**Comment by oupala** [comment]
Good. When do you plan to release a new version of the module?

As a workaround, I found that you can start the field by entering a cr/lf so that it is deleted and everything goes well.

```csv
dcterms:identifier,dcterms:title,dcterms:bibliographicCitation
1503,title string 1,"
bib string 1
bib string 2
bib string 3"
```

Nevermind for the second bug ("br is not rendered as is in the default template"), I can't reproduce it...

**Comment by zerocrates** [comment]
The fix is in [version 2.1.2](https://github.com/omeka-s-modules/CSVImport/releases/download/v2.1.2/CSVImport-2.1.2.zip), out now.

It will be listed on the omeka.org list of modules tomorrow.

**Comment by oupala** [comment]
I've just updated the module, I re-imported all my data and things seems to go well.

Thanks a lot for releasing a new version so quickly!

I think it is possible to close this issue.
